docs/excel/custom-functions-dynamic-arrays.md

```diff
@@ -0,0 +1,58 @@
+---
+ms.date: 12/16/2019
+description: Return a dynamic array from your custom function in an Office Excel add-in.
+title: Return a dynamic array from your custom function
+localization_priority: Normal
+---
+
+# Return a dynamic array from your custom function
+
+You can return multiple results from your custom function to make it a dynamic array formula. Multiple results will spill into neighboring cells. You can return results that only spill down, to the right, or both down and to the right.
+
+To create a custom function that is a dynamic array formula, it must return a two-dimensional array of values. The following example shows how to return a dynamic array that spills down.
+
+```javascript
```

Comment:
My only qualm about this dynamic array thing is that it is hard to understand without seeing it actually work. Is there a chance we can add .gifs or a call out to try this in a Script Lab snippet? Personally I just think they are really confusing. If we think that this article will only be read by people who are already power users who understand what Dynamic Arrays are, then I would leave it.